src/fides/api/ctl/database/seed.py

```diff
@@ -0,0 +1,272 @@
+
+CONFIG = get_config()
+FIDESOPS_AUTOGENERATED_CLIENT_KEY = "fides_autogenerated_client"
+FIDESOPS_AUTOGENERATED_STORAGE_KEY = "fidesops_autogenerated_storage_destination"
+DEFAULT_ACCESS_POLICY = "default_access_policy"
+DEFAULT_ACCESS_POLICY_RULE = "default_access_policy_rule"
+DEFAULT_ERASURE_POLICY = "default_erasure_policy"
+DEFAULT_ERASURE_POLICY_RULE = "default_erasure_policy_rule"
+DEFAULT_ERASURE_MASKING_STRATEGY = "hmac"
+
+
+def filter_data_categories(
+    categories: List[str], excluded_categories: List[str]
+) -> List[str]:
+    """
+    Filter data categories and their children out of a list of categories.
+
+    We only want user-related data categories, but not the parent category
+    We also only want 2nd level categories, otherwise there are policy conflicts
+    """
+    user_categories = [
+        category
+        for category in categories
+        if category.startswith("user.") and len(category.split(".")) < 3
+    ]
+    if excluded_categories:
+        duplicated_categories = [
+            category
+            for excluded_category in excluded_categories
+            for category in user_categories
+            if not category.startswith(excluded_category)
+        ]
+        default_categories = {
+            category
+            for category in duplicated_categories
+            if duplicated_categories.count(category) == len(excluded_categories)
+        }
+        return sorted(list(default_categories))
+    return sorted(user_categories)
+
+
+async def load_default_dsr_policies() -> None:
+    """
+    Checks whether DSR execution policies exist in the database, and
+    inserts them to target a default set of data categories if not.
+    """
+    with sync_session() as db_session:
+
+        client = ClientDetail.get_by(
+            db=db_session,
+            field="fides_key",
+            value=FIDESOPS_AUTOGENERATED_CLIENT_KEY,
+        )
+        if not client:
+            client, _ = ClientDetail.create_client_and_secret(
+                db=db_session,
+                client_id_byte_length=CONFIG.security.oauth_client_id_length_bytes,
+                client_secret_byte_length=CONFIG.security.oauth_client_secret_length_bytes,
+                fides_key=FIDESOPS_AUTOGENERATED_CLIENT_KEY,
+            )
+
+        client_id = client.id
+
+        # By default, include all categories *except* those related to a user's
+        # financial, payment, and credentials data. These are typically not
+        # included in access and erasure requests as they are covered by other
+        # compliance programs (e.g. legal, tax, security) and most
+        # organizations need to be extra careful about how these are used -
+        # especially for erasure! Therefore, a safe default for "out of the
+        # box" behaviour is to exclude these
+        excluded_data_categories = [
+            "user.financial",
+            "user.payment",
+            "user.credentials",
+        ]
```

Comment:
Added this comment @ThomasLaPiana @seanpreston

Reply:
ty 🙏